app/utils/cookies.js

```diff
@@ -133,6 +138,43 @@ function deleteSessionVariable (req, key) {
   }
 }
 
+/**
+ * @param {Request} req
+ * @returns {string}
+ */
+function findSessionChargeToDelete (req) {
+  const now = epochSecondsNow()
+  const yesterday = now - 86400
+  const ninetyMinutesAgo = now - 5400
+
+  const chargesSortedByAccessedDate = getChargesOnSession(req)
+    .map(chargeId => {
+      return {
+        chargeId,
+        state: getSessionChargeState(req, chargeId)
+      }
+    })
+    .sort(({ state: a }, { state: b }) => a.accessedAt - b.accessedAt)
+
+  logger.info(`User session contains ${chargesSortedByAccessedDate.length} charges`)
+
+  const leastRecentlyAccessedChargeCreatedMoreThanTwentyFourHoursAgo = chargesSortedByAccessedDate.find(charge => charge.state.createdAt < yesterday)
+
+  if (leastRecentlyAccessedChargeCreatedMoreThanTwentyFourHoursAgo) { return leastRecentlyAccessedChargeCreatedMoreThanTwentyFourHoursAgo.chargeId }
```

Comment:
Egyption brackets, please:

```javascript
if (leastRecentlyAccessedChargeCreatedMoreThanTwentyFourHoursAgo) {
    leastRecentlyAccessedChargeCreatedMoreThanTwentyFourHoursAgo.chargeId
}
```

(Same goes for the lines below.)

Reply:
done